vividus-plugin-web-app-to-rest-api/src/main/resources/org/vividus/bdd/steps/integration/browser-cookies.ftl

```diff
@@ -69,7 +69,9 @@
                         ${cookie.path}
                     </td>
                     <td>
-                        ${cookie.expiryDate?datetime}
+                        <#if cookie.expiryDate??>
+                            ${cookie.expiryDate?datetime}
+                        </#if>
```

Comment:
Shouldn't you put some value if no expiry date set? What does Chrome show?

Reply:
Good catch, as rfc says If a cookie has neither the Max-Age nor the Expires attribute, the user agent will retain the cookie until "the current session is over" (as defined by the user agent). Took default value from chrome